### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```
Comment (viktoria-rybenchuk) [inline app/main.py]: place each parameter on a new line include self

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
```
Comment (viktoria-rybenchuk) [inline app/main.py]: fix accrording to comment above

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,50 @@
...
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float or int:
```
Comment (viktoria-rybenchuk) [inline app/main.py]: specify type annotations for cars

Comment (viktoria-rybenchuk) [inline app/main.py]:
```suggestion
    def serve_cars(self, cars: list) -> float | int:
```

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,50 @@
...
+        income = 0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+
```
Comment (viktoria-rybenchuk) [inline app/main.py]: remove blank line

File: app/main.py [hunk 5]
```diff
@@ -1,8 +1,50 @@
...
+                self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> int or float:
```
Comment (viktoria-rybenchuk) [inline app/main.py]:
```suggestion
    def calculate_washing_price(self, car: Car) -> int | float:
```

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 6]
```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```
Comment (pavlejviki) [inline app/main.py:6]:
```suggestion
            brand: str
       ) -> None:
```

File: app/main.py [hunk 7]
```diff
@@ -1,8 +1,51 @@
...
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
```
Comment (pavlejviki) [inline app/main.py:19]:
```suggestion
            count_of_ratings: int
        ) -> None:
```

**Review by pavlejviki — APPROVED** [review]
Well done!

**Pull request closed without merging**